app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+{/\* \* Copyright © 2023 United States Government as represented by the
+
+- Administrator of the National Aeronautics and Space Administration.
+- All Rights Reserved.
+-
+- SPDX-License-Identifier: Apache-2.0 \*/}
+
```

Comment:
We've been omitting the license header from MDX files.

```suggestion
```

Reply:
Removed.

Reply:
It's still there.

Reply:
I haven't pushed these changes yet.